insalan/tickets/views.py

```diff
@@ -121,3 +142,104 @@ def generate_pdf(request: HttpRequest, token: str) -> JsonResponse:
 
     pdf = TicketManager.generate_ticket_pdf(ticket)
     return HttpResponse(pdf, content_type='application/pdf')
+
+@api_view(["POST"])
+@permission_classes([IsAdminUser])
+def pay(request: HttpRequest) -> JsonResponse:
+    """
+        This view is used to :
+        - Mark as paid a registration from it's type and id
+        - Create a ticket for the user that paid
+        - Send an email to the user with the ticket
+    """
+    data = request.data
+    print(data)
+    if 'type' not in data or 'id' not in data:
+        return JsonResponse({'err': _("Type ou id manquant")},
+                            status=status.HTTP_400_BAD_REQUEST)
+
+    if data['type'] == 'player':
```

Comment:
Since this is Python, this if chain needs a guard else to fail if the type doesn't match